contracts/proxy/UpgradableProxy.sol

```diff
@@ -0,0 +1,24 @@
+pragma solidity ^0.5.17;
```

Comment:
Both `Proxy.sol` and `UpgradableProxy.sol` are slightly different from their [OZ counterparts](https://github.com/OpenZeppelin/openzeppelin-contracts/tree/master/contracts/proxy). 
I'd recommend using their proxy unless there is a strong argument against it (like your contracts are already deployed and are incompatible on storage layout level)

Reply:
I would agree with you if not tomorrow's deployment.